tests/e2e/lib.sh

```diff
@@ -177,8 +181,10 @@ deploy_stackrox_operator() {
     fi
 }
 
+# shellcheck disable=SC2120
```

Comment:
This should go away, when `deploy_central` is called with the namespace argument right?

Reply:
Yes, I believe it is not required anymore, removed it.

Reply:
There were some more occurrences of this "ignore", would it be possible to remove them as well?

Reply:
Good point, I have removed a few more which seemed safe (i.e. to not cause shellcheck issues in other shell scripts). Hope I haven't removed too many -- didn't want to touch other scripts at this point.

Reply:
As mentioned offline already, removing them requires a bit more analysis. Happy to check this in a separate PR.